apps/backend/src/eventHandlers/logging.ts

```diff
@@ -223,15 +223,15 @@ export default function createHandler() {
               );
 
               description = `Selected instruments: ${instruments?.map((instrument) => instrument.name).join(', ')} is attached to technique: ${technique?.name}`;
-            }
 
-            await eventLogsDataSource.set(
-              event.loggedInUserId,
-              event.type,
-              json,
-              '',
-              description
-            );
+              await eventLogsDataSource.set(
+                event.loggedInUserId,
+                event.type,
+                json,
+                obj[0].techniqueId,
```

Comment:
If this id not a string already we must need to convert it to a string.

Reply:
Since obj is from JSON.parse(), its of type any. It's getting converted to string type when getting passed to this method call.